example.config.toml

```diff
@@ -80,3 +80,12 @@ temperature_monitoring = true # default true
   #   binary = 'C:\Program Files\storcli\storcli64.exe'
   # Enable on Linux:
   binary = '/opt/storcli/sbin/storcli64'
+
+# Settings for the jobmon wrapper for the  monitoring of jobs
+[jobmon]
+  # Path to spool dir
+  #   spool_dir = 'C:\ProgramData\cagent\jobmon' # Windows
+  spool_dir = '/var/lib/cagent/jobmon' # Linux
```

Comment:
what about darwin? `/var/lib` is a system-only directory. 
As I can see, you don't want tmp directory, so it should be something like `~/Library/Application Support/com.cloudradar.cagent/jobmon`

Reply:
Thanks, nice catch.

Reply:
Fixed in a last commit